api/migrations/versions/0197_creating_service_template_when_a_.py

```diff
@@ -0,0 +1,59 @@
+"""
```

Comment:
@andrewoh531 I wonder if you should rename the migration file. It was truncated and doesn't really explain what it was intending to do.

Reply:
Sure